_posts/2018-11-16-JOIN-2018.md

```diff
@@ -0,0 +1,534 @@
+
+****
+
+<img class="image fit" src="{{ '/img/join-2018/join-crowd.jpg' | prepend: site.baseurl }}" alt="Join 2018 crowd" width="60%" />
+
+# Talks & Workshops
+
+* [Frederick Bousson: Unbeatable at Connect 4 with Wearables and Computer Vision](#unbeatable-at-connect-4-with-wearables-and-computer-vision-by-frederick-bousson)
+* [Tom Verelst: Testimonial TVH MyAssetPlanner](#testimonial-tvh-myassetplanner-by-tom-verelst)
+* [Dieter Hubau: Visualizing Mandelbrot fractals using Riff and Spring Cloud Function](#visualizing-mandelbrot-fractals-using-riff-and-spring-cloud-function-by-dieter-hubau)
+* [Pieter Van Hees: Testimonial HZIV](#testimonial-hziv-by-pieter-van-hees)
+* [Julia Müller: The 10 worst mistakes your product owner can make](#the-10-worst-mistakes-your-product-owner-can-make-by-julia-müller)
+* [Bart Blommaerts: Innovation without asking permission](#innovation-without-asking-permission-by-bart-blommaerts)
+
+## Unbeatable at Connect 4 with Wearables and Computer Vision by [Frederick Bousson](https://twitter.com/fbousson){:target="_blank"}
+
+<span class="image left"><img class="p-image" alt="Frederick Bousson" src="/img/join-2018/speaker-bousson.jpg"></span>
+
+Frederick Bousson kicked off the technology track with his talk about wearables.
+
+Wearables are becoming more mainstream every day as proven by the prevalence of fitness trackers and smart watches.
+Smart glasses on the other hand are still very rare.
+Mainly because they still look awful.
+But improvement is on its way here as well: better looking sets are starting to appear like the [Vuzix Blade](https://www.vuzix.com/products/blade-smart-glasses){:target="_blank"}.
+This is important as humans are very vision-oriented: 90% of the information on the internet is processed via the eyes.
+
+### Connect 4
+To really demonstrate the power of smart glasses, Frederick was looking for an interesting case he could use as a demonstration.
+The result of this quest was a way to help you become unbeatable at Connect 4.
+
+Connect 4 is a "solved game", which means that there's an algorithm that can't be defeated (provided you get to go first).
+Frederick went hunting in the Open Source community and discovered a program written by [Brian Borowski](http://www.brian-borowski.com/) which contained this algorithm. 
+
+4 years ago, [Scott Bouloutian](https://github.com/ScottBouloutian/OpenCV-Connect-Four-AI){:target="_blank"} added visual processing code on top of that and yet another person ported the result to Google Glass.
+Nice to see how, with open source, you have people who build upon each others' work.
+
+### Image processing
+If you want to have your smart glasses solve your game of Connect 4, you'll need to do some image processing.
+And since the game revolves around colour which means your program doesn't just need to see, it has to reason about these colors as well.
```

Comment:
```suggestion
And since the game revolves around colour which means your program doesn't just need to see, it has to reason about these colours as well.
```